changelogs/fragments/859-s3_object.yml

```diff
@@ -0,0 +1,7 @@
+minor_changes:
+- aws_s3 - The ``aws_s3`` module has been renamed to ``s3_object``
+  (https://github.com/ansible-collections/amazon.aws/pull/869).
+deprecated_features:
+- s3_object - Support for creation and deletion of S3 buckets has been deprecated.  Please use the
```

Comment:
Doesn't need a redirect in meta/runtime as well?

Reply:
Yup, I forgot to git add it :)